src/components/Header.tsx

```diff
@@ -4,11 +4,11 @@ import NewContactButton from './NewContactButton';
 
 import '../styles/Header.scss';
 
-const logoDesktop = './images/logo-desktop.svg';
+const logoDesktop = '/images/logo-desktop.svg';
 const logoMobile = './images/logo-mobile.svg';
```

Comment:
don't you need to remove the dot here as well?

Reply:
removed the dot, thank you.